app/views/_build-timestamp.html

```diff
@@ -0,0 +1 @@
+<relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp>
```

Comment:
seems excessive to have a partial that just adds a directive, what was the reason for this?